src/Components/List/List.tsx

```diff
@@ -1 +1,32 @@
-export {}
\ No newline at end of file
+import React, { ReactNode } from 'react'
+import './List.css'
+
+type ListProps = {
```

Comment:
I'm wondering if we made this an interface in App if we could then export it from App and import it to be used here, instead of retyping this.  Could make the code more DRY.